mirrord-config/src/network.rs

```diff
@@ -10,18 +10,25 @@ use crate::{
     util::{MirrordToggleableConfig, ToggleableConfig},
 };
 
+/// Controls mirrord network operations.
 #[derive(MirrordConfig, Deserialize, Default, PartialEq, Eq, Clone, Debug)]
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(deny_unknown_fields)]
 #[config(map_to = NetworkConfig)]
 pub struct NetworkFileConfig {
+    /// Mode of operation for incoming network requests in mirrord, supports `mirror` or `steal`:
+    ///
+    /// - `mirror`: mirror requests from a Kubernetes container to the local process;
+    /// ` `steal`: redirects requests from the local process;
```

Comment:
to the local process*

Reply:
Should be :
- `mirror`: mirror incoming requests to the remote pod to the local process
- `steal`: redirect incoming requests to the remote pod to the local process